stages/02_download_data.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+mkdir -p ./download
+
+cat list/target_links.txt | xargs -I {} wget -P download/  -nH --cut-dirs=2 -r -l 1 -A '*.ism' 'https://dude.docking.org/targets/{}'
```

Comment:
Add a
```shell
# TODO Only downloading *.ism files for now.
```
comment so that it is clear that this is a subset of the entire dataset.

Reply:
Resolved in this commit: f1f8b6cffde84b52291b3a9ed1c90f047ba89bec